ethereum/contracts/interfaces/relayer/IWormholeRelayer.sol

```diff
@@ -158,26 +445,71 @@ interface IWormholeRelayerSend is IWormholeRelayerBase {
         address newDeliveryProviderAddress
     ) external payable returns (uint64 sequence);
 
+    /**
+     * @notice Returns the price to request a relay to chain 'targetChain', using the default delivery provider
+     * 
+     * @param targetChain destination chain, in Wormhole Chain ID format
```

Comment:
I don't think we want to introduce "destination" as additional verbiage - let's just stick to target - the parameter itself is sufficiently clear, imho